rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -697,63 +697,6 @@ stream {
     {{ end }}
 
     error_log  {{ $cfg.ErrorLogPath }};
-
```

Comment:
why aren't you removing the whole stream (https://github.com/kubernetes/ingress-nginx/pull/3197/files#diff-980db9e4b88704f12338bd074839f94eR690) block?

Reply:
because I want to refactor the SSL passthrough feature to use the stream section, removing the go proxy code.